### PR TITLE
Add doc page on sysvar accounts

### PR DESCRIPTION
**Pull request by CriesofCarrots**
#### Problem
With only in-code documentation at present, sysvars are not very obvious and accessible for developers to use. (And also there is no where to add these important details: https://github.com/solana-labs/solana/pull/13120#issuecomment-718048769)

#### Summary of Changes
- Add sysvars page to docs.solana.com Developing section

### CODE REVIEW

File: docs/src/apps/sysvars.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+To include sysvar data in program operations, pass the sysvar account address in
+the list of accounts in a transaction. The account can be read in your
+instruction processor like any other account. Access to sysvars is always
+*readonly*.
+
+## Clock
+
+The Clock sysvar contains data on cluster time, including the current slot,
+epoch, and estimated wall-clock timestamp. It is updated every slot.
```
Comment (t-nelson) [inline docs/src/apps/sysvars.md:19]:
```suggestion
epoch, and estimated wall-clock UNIX timestamp. It is updated every slot.
```

File: docs/src/apps/sysvars.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+- Address: `SysvarC1ock11111111111111111111111111111111`
+- Layout: [Clock](https://docs.rs/solana-sdk/VERSION_FOR_DOCS_RS/solana_sdk/sysvar/clock/struct.Clock.html)
+
+## EpochSchedule
+
+The EpochSchedule sysvar contains epoch scheduling data that is set in genesis,
```
Comment (t-nelson) [inline docs/src/apps/sysvars.md:26]: Should we mention something about "epoch schedule" being aka "leader schedule"

File: docs/src/apps/sysvars.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+The SlotHistory sysvar contains a bitvector of slots present over the last
+epoch. It is updated every slot.
+
+- Address: `SysvarS1otHistory11111111111111111111111111`
+- Layout: [SlotHistory](https://docs.rs/solana-sdk/VERSION_FOR_DOCS_RS/solana_sdk/sysvar/slot_history/struct.SlotHistory.html)
+
+## StakeHistory
+
+The StakeHistory sysvar contains the history of stake activations and
```
Comment (t-nelson) [inline docs/src/apps/sysvars.md:85]: global? cluster-wide? stake activations

**Review by t-nelson — COMMENTED** [review]
Looks great!  Just a few clarifying nits. It's going to be nice having this to link to!

File: docs/src/apps/sysvars.md (hunk 2 above)
Comment (CriesofCarrots) [reply docs/src/apps/sysvars.md:26]: Do you mean clarify that epoch schedule is *not* the same as leader schedule?

Comment (t-nelson) [reply docs/src/apps/sysvars.md:26]: Whoops! Great typo there... yeah exactly.

**Review by garious — APPROVED** [review]
Awesome, thank you!

File: docs/src/apps/sysvars.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
+---
+title: Sysvar Cluster Data
+---
+
+Solana exposes a variety of cluster state data to programs via
+[`sysvar`](terminology.md#sysvar) accounts. These accounts are populated at
+known addresses published along with the account layouts in the
+[`solana-sdk`](https://docs.rs/solana-sdk/VERSION_FOR_DOCS_RS/solana_sdk/sysvar/index.html),
```
Comment (mvines) [inline docs/src/apps/sysvars.md:8]: Post-merge review!  Please use `solana-program` instead of `solana-sdk`, which is where these now live as of 1.4

Comment (CriesofCarrots) [reply docs/src/apps/sysvars.md:8]: Okay, will update

**Pull request merged**